Sources/SwifterSwift/SwiftStdlib/ArrayExtensions.swift

```diff
@@ -161,5 +161,34 @@ public extension Array where Element: Equatable {
             }
         }
     }
+}
 
+extension Array where Element: Hashable {
+    
+    public func filtered<E: Equatable>(duplication path: KeyPath<Element, E>) -> [Element] {
+        return reduce(into: [Element]()) { (result, e) in
```

Comment:
Don't use `e` as a variable name, make it more meaningful.
```suggestion
        return reduce(into: [Element]()) { (result, element) in
```